File: Assignment 4 copy/Person.java
public class Person {

	private String name;
	private String address;
	private String phone;
	
	
	
	Person(String newName, String address, String phone) {
		
		this.name = newName;
		this.address = address;
		this.phone = phone;
	}
	
	void setName(String name) {//sets the value of the name variable to whatever is type into it by the user
		
		this.name = name;
		//this keyword refers to current object in a method or constructor and most common use of the keyboard is to avoid confusion between class attributes and parameters of the same name	
		
	}
	
	String getName() {//returns what is currently stored in the name variable as a string object
		
		
		return(this.name);
		
	}
	void setPhone(String phone) {
		
		this.phone = phone;
		
		
	}
	
	String getPhone() {
		
		return(this.phone);
		
		
	}
	
	public String toString() {//returns string representation of the object
		return "\nName: " + name +"\nAddress: "+address+ "\nPhone: " + phone;
		
		
	}

	
	

}

    

File: Assignment 3 backup copy/Person.py
class Person:

    def __init__(self,newName = "none",address = "none",phone = "999-999-9999"): # class constructor
        self.name = newName
        self.phone = phone
        self.address = address
        self.next = None
        self.prev = None

    def __eq__(self,other):
        if (isinstance(other, Person)):
            return(self.getphone()==other.getphone())
        if(isinstance(other,str)):
            return(self.getphone==other)
        else:
            return False
        


    def __str__(self):
        return "Name" + self.name + "address"+ self.address +"phone" + self.phone
    
    
    def setName(self,name):
        self.name = name

    def setAddress(self, address):
        self.address = address

    def setPhone(self,phone):
        self.phone = phone
    


    def getName(self):
        return(self.name)
    
    def getAddress(self):
        return(self.address)
    
    def setPhone(self):
        return(self.phone)
    
    def getPhone(self):
        return(self.phone)
    
    
    
    








    





        
